R/linter_tags.R

```diff
@@ -157,17 +157,26 @@ rd_taglist <- function() {
 #' @noRd
 rd_linterlist <- function() {
   linters <- available_linters()
-  linter_names <- sort(linters[["linter"]])
+  linter_names <- platform_independent_sort(linters[["linter"]])
 
   c(
     "\\section{Linters}{",
     "The following linters exist:",
     "\\itemize{",
     vapply(linter_names, function(linter_name) {
-      tags <- sort(linters[["tags"]][[match(linter_name, linters[["linter"]])]])
+      tags <- platform_independent_sort(linters[["tags"]][[match(linter_name, linters[["linter"]])]])
       paste0("\\item{\\code{\\link{", linter_name, "}} (tags: ", toString(tags), ")}")
     }, character(1L)),
     "}", # itemize
     "}" # section
   )
 }
+
+platform_independent_sort <- function(x) {
+  if (!identical(sort(c("a", "B")), c("a", "B"))) {
+    stop("Please run in an English-like locale, e.g. LC_COLLATE=en_US")
+  }
+  # see #923 -- some locales ignore _ when running sort(), others don't.
```

Comment:
weird that this triggers `commented_code_linter`. seems like a bug.

Reply:
Probably because `see #...` is parseable and the text after "#" contains code-ish chars (`sort()`).
`see issue #...` should suppress the lint.

Reply:
still sounds like a false positive to me. filing for followup